### PR TITLE
Prevent the possibility of starting multiple concurrent experiments

### LINKED ISSUE #6335: Possible to start concurrent simulations when running experiment
The run experiment button seems to be enabled if you select a simulation mode during execution
You are then able to run concurrent experiments.

Some options:
- Make rundialog modal
- Avoid enable the button if the text on the button is “Experiment running..”
- Don’t enable button on simulation mode change

### PR DESCRIPTION
**Pull request by larsevj**
**Issue**
Resolves #6335 

## Pre review checklist

- [x] Read through the code changes carefully after finishing work
- [x] Make sure tests pass locally (after every commit!)
- [x] Prepare changes in small commits for more convenient review (optional)
- [x] PR title captures the intent of the changes, and is fitting for release notes.
- [ ] Updated documentation
- [ ] Ensured that unit tests are added for all new behavior (See 
    [Ground Rules](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md#ground-rules)),
    and changes to existing code have good test coverage.

## Pre merge checklist
- [x] Added appropriate release note label
- [ ] Commit history is consistent and clean, in line with the [contribution guidelines](https://github.com/equinor/ert/blob/main/CONTRIBUTING.md).

### CODE REVIEW

**Comment by larsevj** [comment]
Not sure if this is the best way to go about this, but seems to be a way. Removed some of the fragility of the check by introducing a Global variable for the message, but unsure if this is desired.

File: src/ert/gui/simulation/simulation_panel.py [hunk 1]
```diff
@@ -217,3 +220,5 @@ def toggleSimulationMode(self):
     def validationStatusChanged(self):
         widget = self._simulation_widgets[self.getCurrentSimulationModel()]
         self.run_button.setEnabled(widget.isConfigurationValid())
+        if self.run_button.text() == EXPERIMENT_IS_RUNNING_BUTTON_MESSAGE:
```
Comment (andreas-el) [inline src/ert/gui/simulation/simulation_panel.py:223]: Would it make sense to setEnabled() only once checking both conditions? You can see which you like best.

File: src/ert/gui/simulation/simulation_panel.py [hunk 2]
```diff
@@ -216,4 +219,10 @@ def toggleSimulationMode(self):
...
     def validationStatusChanged(self):
         widget = self._simulation_widgets[self.getCurrentSimulationModel()]
-        self.run_button.setEnabled(widget.isConfigurationValid())
+        if (
+            self.run_button.text() == EXPERIMENT_READY_TO_RUN_BUTTON_MESSAGE
+            and widget.isConfigurationValid()
+        ):
+            self.run_button.setEnabled(True)
+        else:
+            self.run_button.setEnabled(False)
```
Comment (andreas-el) [inline src/ert/gui/simulation/simulation_panel.py:228]: Alternatively a more compact variant:
```
self.run_button.setEnabled(
    self.run_button.text() == EXPERIMENT_READY_TO_RUN_BUTTON_MESSAGE
    and widget.isConfigurationValid()
)
```

**Comment by andreas-el** [comment]
Could you provide a more descriptive PR header?

**Review by andreas-el — APPROVED** [review]
Nice! 🚀

**Pull request merged**